packages/contracts/package.json

```diff
@@ -10,11 +10,13 @@
         "build": "./scripts/buildVerifiers.sh && yarn compile && npx hardhat typechain && tsc",
         "compile": "npx hardhat compile",
         "test": "npx hardhat test --no-compile",
-        "lint": "prettier --write ."
+        "lint": "prettier --write .",
+        "coverage": "npx hardhat coverage"
     },
     "dependencies": {
         "@unirep/circuits": "1.0.0",
         "@unirep/crypto": "1.0.0",
+        "crypto-js": "^4.1.1",
```

Comment:
Do we use `crypto-js` here?

Reply:
We need it for `solidity-coverage`. I moved it to devDependencies. Good catch thanks!